### DISCUSSION TITLE
멤버 모집을 위해 애써주신 스터디 3기 리더분들께 감사의 마음을 전해요 ❤️

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#647] [칭찬감사]
스터디 3기의 리더이신 @SamTheKorean 님과 @TonyKim9401 님께 감사의 마음을 전합니다!
기존 대기자와 신규 신청자가 혼재된 상황에서 디스코드 서버나 깃허브 팀에 가입 안 하신 분들 일일이 다 챙기시느라 진짜 고생을 많이 하셨습니다. 😭
리더님들의 노고 덕분에 이전 기수의 두 배가 넘는 분들과 함께 새로운 기수를 시작할 수 있어서 너무 기쁩니다!

**Comment by obzva** [comment]
고생 많으셨습니다 두 분!!

**Comment by SamTheKorean** [comment]
코치님들이 함께 많이 기여해주셔서 스터디가 순조롭게 시작되고 있는 것 같습니다! 기여에 감사드립니다! @obzva @HC-kang @EgonD3V

**Comment by TonyKim9401** [comment]
@DaleSeo 님께 스터디 리더로 참여할 수 있는 좋은 기회를 주셔서 감사합니다 
@SamTheKorean 님 처음이라 모르는것이 많아 리더로써 역할 분배를 적절히 해주시고 의견 공유해주서서 감사합니다.
아울러 온보딩하신 스터디 멤버들의 스터디 진행 방향에 대해 코멘트 및 가이드 남겨주신 코치분들께도 감사의 말씀 드립니다.
3기 스터디 진행 동안 잘 부탁드립니다.
@obzva @HC-kang @EgonD3V 

모두 감사드립니다!

**Comment by yolophg** [comment]
리더분들 감사드려요 :).
